docs/preview/guides/migration-guide-v1.0.md

```diff
@@ -0,0 +1,86 @@
+# Migration guide towards v1.0
+Starting from v1.0, there're some major breaking changes. To make it easier for you you migrate towards this new version, we have assembled an migration guide to help you in the process.
+
+- [New Azure SDK](#new-azure-sdk)
```

Comment:
Our get correlation info extension is also "broken" because it no longer works with old SDK

Reply:
Yeah, we could mention that, although, it works with the `ServiceBusReceivedMessage` of the new SDK and since the old SDK's `Message` type is not there anymore... Not sure if it adds more value. But I'll add it!

Reply:
Yes we should add it since it's a breaking change as well - thank you!